Find a functionally equivalent code:android.app.WallpaperManager.clear:COMMENT
Method Modifier: public      
Comment:/**
 * Remove any currently set wallpaper, reverting to the system's built-in
 * wallpaper. On success, the intent {@link Intent#ACTION_WALLPAPER_CHANGED}
 * is broadcast.
 *
 * <p>This method requires the caller to hold the permission
 * {@link android.Manifest.permission#SET_WALLPAPER}.
 *
 * @throws IOException If an error occurs reverting to the built-in
 * wallpaper.
 */

@@ -1,3 +1,3 @@
 {
-    setResource(com.android.internal.R.drawable.default_wallpaper);
+    setStream(openDefaultWallpaper(mContext));
 }

